libs/env/src/env.ts

```diff
@@ -189,9 +189,14 @@ const envDefinitions = {
   },
   SOLANA_VALIDATORS_APP_BASE_URL: {
     parser: stringParser,
-    def: "https://validators-solana.coin.ledger.com/api/v1/validators",
+    def: "https://earn-dashboard.aws.prd.ldg-tech.com/v0/network/solana/validator-details",
```

Comment:
Changes look good, however we need to confirm with infra and/or PTX that this URL is the one we want in prod.